src/contribot/lib/contribot.py

```
@@ -0,0 +1,2046 @@
+        df["num_errors"] = self.summary["num_errors"]
+
+        try:
+            if engine:
+                with engine.connect() as conn:
+                    df.to_sql(table_name, con=conn, index=False, if_exists="append")
+            else:
+                with create_engine(
+                    "mysql+mysqlconnector://opensupplyhub:secret@localhost/opensupplyhub"
+                ).connect() as conn:
+                    df.to_sql(table_name, con=conn, index=False, if_exists="append")
+        except:
```

Ruff (0.15.21)

[error] Do not use bare `except`

(E722)